packages/api-client/tests/test-CumulusApiClient.js

```diff
@@ -26,8 +26,8 @@ test.serial('invokeApi invokes the lambda with the expected Payload and Function
   };
 
   fakeServices.lambda = () => ({
-    invoke: (payloadObject) => {
-      const passedPayload = payloadObject;
+    send: (payloadObject) => {
```

Comment:
Hmm different approach to checking that the lambda service received the correct command/input. Probably ok though as we don't want to reconfigure this whole test if we can avoid it. 

Could this be fragile? 🤔 Maybe not because we're at a very high level and `invokeApi` isn't likely to make more calls to the lambda service in the future...?

Reply:
Right, that last thing, structurally the api-client package shouldn't *ever* be doing anything other than a direct invocation of the API lambda, this feels fairly safe.